### PR TITLE
Cleanup config files

### PR DESCRIPTION
**Pull request by n-rodriguez**
This PR remove white spaces in plugins config files.

### CODE REVIEW

**Comment by myii** [comment]
@n-rodriguez I've done a brief eyeball check and this looks like a significant and excellent contribution.  The only comment I have is regarding the use of Jinja whitespace control, specifically doubling up the `-`, i.e. `{%- ... -%}`.  References:

* Issue raised: https://github.com/saltstack-formulas/postgres-formula/issues/241
* Resolved by: https://github.com/saltstack-formulas/postgres-formula/pull/244
* Upstream issue: https://github.com/saltstack/salt/issues/35398
  - This issue was closed but was reopened after some discussions
* Another issue: https://github.com/saltstack-formulas/openssh-formula/issues/148

Basically, as a rough guideline to minimise formula breakages, limit whitespace control to only a single `-`, usually `{%- ... %}`.

**Comment by n-rodriguez** [comment]
> Basically, as a rough guideline to minimise formula breakages, limit whitespace control to only a single `-`, usually `{%- ... %}`.

Yes, I understand this kind of concern, but actually the '-' I added are here to remove empty lines at the top of files. The goal was to trim whitespaces so I was very careful to not change the logic implemented.

**Comment by myii** [comment]
@n-rodriguez I fully understand.  Have you tried setting the following in your `master` config?

```yaml
jinja_trim_blocks: True
```

I was amazed at how aggressive this is!  I spent a fair while with this when arguing things in the [upstream issue](https://github.com/saltstack/salt/issues/35398) and I found it broke (and I mean broke) so many formulas.  I've asked for this to be improved, where we can [override this setting as formula writers](https://github.com/saltstack/salt/issues/35398#issuecomment-457240072), but until then we're going to have to be cautious about "excessive" whitespace control.

Ultimately, it's your PR and this is only a general suggestion, nothing enforced.

**Comment by n-rodriguez** [comment]
Fixed!

File: collectd/files/apache.conf [hunk 1]
```diff
@@ -9,29 +8,28 @@
...
-    {%- endif -%}
-    {% if instance.server is defined and instance.server %}
-                Server "{{ instance.server }}"
-    {% endif %}
-        </Instance>
+  <Instance "{{ instance.name }}">
+    URL "{{ instance.url }}"
+{%- if instance.user is defined and instance.user %}
+    User "{{ instance.user }}"
+{%- endif -%}
```
Comment (myii) [inline collectd/files/apache.conf]: `{%- ... -%}`

File: collectd/files/apache.conf [hunk 2]
```diff
@@ -9,29 +8,28 @@
...
+{% if instance.pass is defined and instance.pass %}
+    Password "{{ instance.pass }}"
+{%- endif -%}
```
Comment (myii) [inline collectd/files/apache.conf]: [Likewise](https://github.com/saltstack-formulas/collectd-formula/pull/141/files#r277121352).

File: collectd/files/apache.conf [hunk 3]
```diff
@@ -9,29 +8,28 @@
...
+{% if instance.verifypeer is defined and instance.verifypeer %}
+    VerifyPeer {{ instance.verifypeer }}
+{%- endif -%}
```
Comment (myii) [inline collectd/files/apache.conf]: [Likewise](https://github.com/saltstack-formulas/collectd-formula/pull/141/files#r277121352).

File: collectd/files/apache.conf [hunk 4]
```diff
@@ -9,29 +8,28 @@
...
+{% if instance.verifyhost is defined and instance.verifyhost %}
+    VerifyHost {{ instance.verifyhost | lower }}
+{%- endif -%}
```
Comment (myii) [inline collectd/files/apache.conf]: [Likewise](https://github.com/saltstack-formulas/collectd-formula/pull/141/files#r277121352).

File: collectd/files/apache.conf [hunk 5]
```diff
@@ -9,29 +8,28 @@
...
+{% if instance.cacert is defined and instance.cacert %}
+    CACert "{{ instance.cacert }}"
+{%- endif -%}
```
Comment (myii) [inline collectd/files/apache.conf]: [Likewise](https://github.com/saltstack-formulas/collectd-formula/pull/141/files#r277121352).

**Comment by myii** [comment]
@n-rodriguez Just the last ones that slipped through... after these, I'll leave you alone, promise!

**Comment by n-rodriguez** [comment]
done
